polkadot.js

```diff
@@ -0,0 +1,727 @@
+        "code": "const { ApiPromise, WsProvider } = require('@polkadot/api');\n\nasync function main() {\nconst wsProvider = new WsProvider('wss://rpc.polkadot.io');\nconst api = await ApiPromise.create({ provider: wsProvider });\nconst address = 'FAKEADDRESS';\nconst balance = await api.query.system.account(address);\nconsole.log(`Balance of ${address}: ${balance}`);\n}"
+      },
+      {
+        "id": 6,
+        "imageUrl": "https://polkadot.network/meta/og-image-staking.png",
+        "title": "The art of staking",
+        "details": "Staking involves locking up your DOT tokens to help secure the network and earn rewards. Validators and nominators contribute to the stability of the ecosystem by staking DOT and supporting validator candidates. Nominators earn rewards proportional to their staked amounts while backing validators."
+      },
+      {
+        "id": 7,
+        "choices": [
+          "Storing and managing your DOT tokens",
+          "Participating in staking and nominating",
+          "Finding yield farming strategies"
+        ],
+        "correctChoice": "Finding yield farming strategies",
+        "title": "Which of these is NOT a function of Polkadot.js?"
+      },
+      {
+        "id": 8,
+        "choices": [
+          "The users of Polkadot",
+          "The validators",
+          "The engineers building new parachains",
+          "All the above"
+        ],
+        "correctChoice": "All the above",
+        "title": "Who contributes to the stability of the ecosystem?"
+      },
+      {
+        "id": 9,
+        "imageUrl": "https://i.imgur.com/YA3ysPy.png",
+        "title": "Power to the people",
+        "details": "Polkadot has a unique governance system that empowers its community to participate in decision-making. By holding DOT tokens, you too can vote on proposals and help shape the future of the Polkadot network. This is a super cool way of getting everybody involved!"
+      },
+      {
+        "id": 10,
+        "imageUrl": "https://i.imgur.com/GT9uGcw.png",
+        "title": "Democracy in action",
+        "details": "Polkadot's referendum system lets DOT holders have a say in important network changes. You can participate in referenda to influence updates and modifications to the ecosystem, including the staking process. It's decentralized decision-making at its best!"
+      },
+      {
+        "id": 11,
+        "title": "Finally... getting started with Polkadot.js. Take a look at the code below to see if you can figure out how it works!",
+        "language": "javascript",
+        "code": "const { ApiPromise, WsProvider } = require('@polkadot/api');\n\nasync function main() {\n// Connect to a local Polkadot node\nconst wsProvider = new WsProvider('wss://rpc.polkadot.io');\nconst api = await ApiPromise.create({ provider: wsProvider });\n\n// Retrieve the latest block number\nconst blockNumber = await api.rpc.chain.getHeader();\nconsole.log(`Latest block number: ${blockNumber.number}`);\n}"
```

Comment:
this is a bad call to action, also given that most of the same code is above.

IMO better here to link to a starting template for desktop and perhaps the NPM instructions they should install into an existing project